src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -25,7 +26,7 @@ public class AzureAppConfigurationOptions
 
         private Dictionary<string, KeyValueWatcher> _changeWatchers = new Dictionary<string, KeyValueWatcher>();
         private List<KeyValueWatcher> _multiKeyWatchers = new List<KeyValueWatcher>();
-        private List<IKeyValueAdapter> _adapters = new List<IKeyValueAdapter>() { new AzureKeyVaultKeyValueAdapter(new AzureKeyVaultSecretProvider()) };
+        private List<IKeyValueAdapter> _adapters = new List<IKeyValueAdapter>() { new AzureKeyVaultKeyValueAdapter(new AzureKeyVaultSecretProvider()), new JsonKeyValueAdapter() };
```

Comment:
This line is a bit long, off screen. Can we do it like

```
adapters = new List {
  a,
  b
};
```

Reply:
Done

---
In reply to: [456047463](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r456047463) [](ancestors = 456047463)

Reply:
Whoops my bracket placement was inconsistent in my suggestion. To be consistent with the rest of the code we would need like

```
  a = new List
  {
    b,
    c
  }
```

---
In reply to: [456090082](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r456090082) [](ancestors = 456090082,456047463)